plugins/net_plugin/net_plugin.cpp

```diff
@@ -512,10 +511,12 @@ namespace eosio {
 
 
       struct chain_info_t {
-         uint32_t      lib_num = 0;
          block_id_type lib_id;
-         uint32_t      head_num = 0;
+         uint32_t      lib_num = 0;
          block_id_type head_id;
+         uint32_t      head_num = 0;
+         block_id_type fork_head_id;
```

Comment:
Why not `forkdb_head_num` and `forkdb_head_id` to be clear? Too verbose?

Reply:
I just went with the old original naming, see https://github.com/EOSIO/eos/blob/v1.0.0/plugins/net_plugin/net_plugin.cpp#L485
The important thing is it is a fork head as opposed to the chain head.